src/main/java/CommunicateCaseHandle.java

```diff
@@ -0,0 +1,134 @@
+import java.util.Scanner;
+public class CommunicateCaseHandle{
+    protected ExceptionsHandle e;
+    public CommunicateCaseHandle(){
+        e = new ExceptionsHandle();
+    }
+    public void dividingLine(){
+        System.out.println("\t__________________________________________________");
+    }
+    public void listHandle(int index, Task[] list){
+        dividingLine();
+        System.out.println("\tHere are the tasks in your list:");
+        for(int i = 0; i < index; i++) {
+            System.out.println("\t" +(i + 1) + "." + list[i].getIdentity() + list[i].getStatusIcon() + " " + list[i].getContent());
+        }
+        dividingLine();
+    }
+    public void taskHandle(String line, Scanner in, Task[] list, int index){
+        int spaceIndex = line.indexOf(" ");
+        if(line.startsWith("todo")){
+            todoHandle(line.substring(spaceIndex + 1), list, index);
+        }else if(line.startsWith("deadline")){
```

Comment:
Need to leave a white space between the if and each else if statement

Reply:
OK